ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -97,6 +98,7 @@ static inline PrjFS_NotificationType KUMessageTypeToNotificationType(MessageType
 
 static bool IsVirtualizationRoot(const char* fullPath);
 static void CombinePaths(const char* root, const char* relative, char (&combined)[PrjFSMaxPath]);
+static const char* GetRelativePath(const char* path, const char* root);
```

Comment:
Naming suggestion: `path` -> `fullPath` for consistency with the other helper functions that operate on full paths.

Reply:
Sounds good.